plugins/modules/sonic_qos_interfaces.py

```diff
@@ -303,6 +312,7 @@
 # sonic# show running-configuration interface Eth 1/5
 # !
 # interface Eth1/5
+#  cable-length 40m
```

Comment:
Not a type. Deletion of cable-length configuration will revert it to default 40m.  Because our current infra does not filter out any default, 40m will show.

Reply:
What do you mean? Is not line 315: cable-length 40m reflecting the default?

Reply:
done